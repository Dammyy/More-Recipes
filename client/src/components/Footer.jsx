import React from 'react';
import { Link } from 'react-router';


const Footer = () => {
  return (
    <footer className="footer">
      <div className="container">
        <div className="row">
          <div className="col-md-6">
            <Link
              to="https://github.com/Dammyy"
            >
              <img src="img/github.png" alt="" />
            </Link>
          </div>
          <div className="col-md-6">
            <span className="text-muted">Copyright Â© More Recipes!</span>
          </div>
        </div>
      </div>
    </footer>

  );
};

export default Footer;
