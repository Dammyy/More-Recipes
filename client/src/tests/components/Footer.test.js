import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/Footer';

describe('Footer component', () => {
  describe('Snapshot', () => {
    test('component should fully render', () => {
      const component = shallow(<Footer />);
      expect(component).toMatchSnapshot();
    });
  });
});
