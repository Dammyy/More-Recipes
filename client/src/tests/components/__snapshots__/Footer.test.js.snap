// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Snapshot component should fully render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col-md-6"
          >
            <a
              href="https://github.com/Dammyy"
              rel="noopener noreferrer"
              target="_blank"
            >
              <img
                alt=""
                src="img/github.png"
              />
            </a>
          </div>
          <div
            className="col-md-6"
          >
            <span
              className="text-muted"
            >
              Copyright © More Recipes!
            </span>
          </div>
        </div>
      </div>,
      "className": "footer",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="row"
        >
          <div
            className="col-md-6"
          >
            <a
              href="https://github.com/Dammyy"
              rel="noopener noreferrer"
              target="_blank"
            >
              <img
                alt=""
                src="img/github.png"
              />
            </a>
          </div>
          <div
            className="col-md-6"
          >
            <span
              className="text-muted"
            >
              Copyright © More Recipes!
            </span>
          </div>
        </div>,
        "className": "container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="col-md-6"
            >
              <a
                href="https://github.com/Dammyy"
                rel="noopener noreferrer"
                target="_blank"
              >
                <img
                  alt=""
                  src="img/github.png"
                />
              </a>
            </div>,
            <div
              className="col-md-6"
            >
              <span
                className="text-muted"
              >
                Copyright © More Recipes!
              </span>
            </div>,
          ],
          "className": "row",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <a
                href="https://github.com/Dammyy"
                rel="noopener noreferrer"
                target="_blank"
              >
                <img
                  alt=""
                  src="img/github.png"
                />
              </a>,
              "className": "col-md-6",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <img
                  alt=""
                  src="img/github.png"
                />,
                "href": "https://github.com/Dammyy",
                "rel": "noopener noreferrer",
                "target": "_blank",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "src": "img/github.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": "a",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <span
                className="text-muted"
              >
                Copyright © More Recipes!
              </span>,
              "className": "col-md-6",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Copyright © More Recipes!",
                "className": "text-muted",
              },
              "ref": null,
              "rendered": "Copyright © More Recipes!",
              "type": "span",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container"
        >
          <div
            className="row"
          >
            <div
              className="col-md-6"
            >
              <a
                href="https://github.com/Dammyy"
                rel="noopener noreferrer"
                target="_blank"
              >
                <img
                  alt=""
                  src="img/github.png"
                />
              </a>
            </div>
            <div
              className="col-md-6"
            >
              <span
                className="text-muted"
              >
                Copyright © More Recipes!
              </span>
            </div>
          </div>
        </div>,
        "className": "footer",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="row"
          >
            <div
              className="col-md-6"
            >
              <a
                href="https://github.com/Dammyy"
                rel="noopener noreferrer"
                target="_blank"
              >
                <img
                  alt=""
                  src="img/github.png"
                />
              </a>
            </div>
            <div
              className="col-md-6"
            >
              <span
                className="text-muted"
              >
                Copyright © More Recipes!
              </span>
            </div>
          </div>,
          "className": "container",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="col-md-6"
              >
                <a
                  href="https://github.com/Dammyy"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <img
                    alt=""
                    src="img/github.png"
                  />
                </a>
              </div>,
              <div
                className="col-md-6"
              >
                <span
                  className="text-muted"
                >
                  Copyright © More Recipes!
                </span>
              </div>,
            ],
            "className": "row",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  href="https://github.com/Dammyy"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <img
                    alt=""
                    src="img/github.png"
                  />
                </a>,
                "className": "col-md-6",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <img
                    alt=""
                    src="img/github.png"
                  />,
                  "href": "https://github.com/Dammyy",
                  "rel": "noopener noreferrer",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "src": "img/github.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                "type": "a",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <span
                  className="text-muted"
                >
                  Copyright © More Recipes!
                </span>,
                "className": "col-md-6",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Copyright © More Recipes!",
                  "className": "text-muted",
                },
                "ref": null,
                "rendered": "Copyright © More Recipes!",
                "type": "span",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
